README.md

```diff
@@ -305,6 +305,15 @@ etc.
 
 ## Building
 
+Binaryen uses git submodules (at time of writing just for gtest), so before you build you will have to initialize the submodules:
+
+```
+git submodule init
+git submodule update
+```
+
+After that you can build with CMake:
+
```

Comment:
Perhaps add a comment about the cmake flag that can disable gtest, as a workaround? That might be useful for packagers where they are building binaryen outside of a git environment.

Reply:
I don't think we ended up adding such a flag.

Reply:
Oh, hmm, maybe we should... anyhow, lgtm here.